README.md

```diff
@@ -157,10 +165,19 @@ The Ethereum common tests are in a submodule. To update you'll want to:
-| Peer to Peer Networking                                                             | Currently we can connect to one of the Ethereum bootnodes, get a list of peers, and add them to a list of known peers. We have fully implemented the modified [kademlia DHT](https://github.com/poanetwork/mana/tree/master/apps/ex_wire/lib/ex_wire/kademlia). <br /><br />We can also successfully perform the encrypted handshake with peer nodes and derive secrets to frame the rest of the messages. We have not yet implemented the ability to send [multi-frame packets](https://github.com/ethereum/devp2p/blob/master/rlpx.md#framing). See Issue [#97](https://github.com/poanetwork/mana/issues/97). |
-| DEVp2p Protocol and Ethereum Wire Protocol                                          | These are partially implemented but need to be completed. See Issue [#166](https://github.com/poanetwork/mana/issues/166) and Issue [#167](https://github.com/poanetwork/mana/issues/167).                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Encoding and Hashing                                                                | The [RLP] encoding protocol and the [Merkle Patricia Tree] data structure are fully implemented.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [Ethereum Virtual Machine] | Our EVM currently passes 100% of the common [VM tests]. We are still discovering subtle differences in our implementation such as the [“vanishing Ether” issue]. This component is for the most part complete.                                                                                                                                                                                                                                                       |
```

Comment:
Is "vanishing Ether" still an issue? Any other additions to current project status welcome.

Reply:
We've handled it so it's not an "issue" anymore. We may be able to remove this subtlety.